homeassistant/components/homekit/type_covers.py

```diff
@@ -37,10 +37,28 @@
     CHAR_TARGET_POSITION,
     CHAR_TARGET_TILT_ANGLE,
     DEVICE_PRECISION_LEEWAY,
+    HK_DOOR_CLOSED,
+    HK_DOOR_CLOSING,
+    HK_DOOR_OPEN,
+    HK_DOOR_OPENING,
     SERV_GARAGE_DOOR_OPENER,
     SERV_WINDOW_COVERING,
 )
 
+DOOR_CURRENT_HASS_TO_HK = {
+    STATE_OPEN: HK_DOOR_OPEN,
+    STATE_CLOSED: HK_DOOR_CLOSED,
+    STATE_OPENING: HK_DOOR_OPENING,
+    STATE_CLOSING: HK_DOOR_CLOSING,
+}
+DOOR_TARGET_HASS_TO_HK = {
+    STATE_OPEN: HK_DOOR_OPEN,
+    STATE_CLOSED: HK_DOOR_CLOSED,
+    STATE_OPENING: HK_DOOR_OPEN,
+    STATE_CLOSING: HK_DOOR_CLOSED,
```

Comment:
Shouldn't this be

```suggestion
    STATE_OPENING: HK_DOOR_OPENING,
    STATE_CLOSING: HK_DOOR_CLOSING,
```

Reply:
HomeKit only has 2 valid states for target door state so this is correct.  I'll add a comment.

Homekit has 5 valid states for current door state (we don't map stopped)